src/load/azext_load/tests/latest/test_load_test.py

```diff
@@ -335,6 +339,128 @@ def test_load_test_create(self, rg, load, vnet):
+        ]
+        # Create load test with autostop disabled thru config file
+        self.cmd(
+            "az load test create "
+            "--test-id {test_id} "
+            "--load-test-resource {load_test_resource} "
+            "--resource-group {resource_group} "
+            '--load-test-config-file "{load_test_config_file}" '
+            "--description {description} "
+            "--display-name {display_name} "
+            "--engine-instance {engine_instance} ",
+            checks=checks,
+        )
+        # Update load test with autostop criteria thru command line arguments
+        self.kwargs.update(
+            {
+                "autostop_error_rate": LoadTestConstants.AUTOSTOP_ERROR_RATE,
+                "autostop_error_rate_time_window": LoadTestConstants.AUTOSTOP_ERROR_RATE_TIME_WINDOW,
+            }
+        )
+        checks = [
+            JMESPathCheck("autoStopCriteria.autoStopDisabled", False),
+            JMESPathCheck("autoStopCriteria.errorRate", LoadTestConstants.AUTOSTOP_ERROR_RATE),
+            JMESPathCheck("autoStopCriteria.errorRateTimeWindowInSeconds", LoadTestConstants.AUTOSTOP_ERROR_RATE_TIME_WINDOW),
+        ]
+        self.cmd(
+            "az load test update "
+            "--test-id {test_id} "
+            "--load-test-resource {load_test_resource} "
+            '--autostop-error-rate {autostop_error_rate} '
+            '--autostop-time-window {autostop_error_rate_time_window} '
+            "--resource-group {resource_group} ",
+            checks=checks,
+        )
+        # Update load test with autostop disabled thru command line arguments
+        self.kwargs.update(
+            {
+                "autostop": LoadTestConstants.AUTOSTOP_DISABLED,
+            }
+        )
+        checks = [
+            JMESPathCheck("autoStopCriteria.autoStopDisabled", True),
+        ]
+        self.cmd(
+            "az load test update "
+            "--test-id {test_id} "
+            "--load-test-resource {load_test_resource} "
+            '--autostop {autostop} '
+            "--resource-group {resource_group} ",
+            checks=checks,
+        )
+        # Update load test with autostop criteria thru config file
+        self.kwargs.update(
+            {
+                "load_test_config_file": LoadTestConstants.LOAD_TEST_CONFIG_FILE_WITH_AUTOSTOP,
+            }
+        )
+        checks = [
+            JMESPathCheck("autoStopCriteria.autoStopDisabled", False),
+            JMESPathCheck("autoStopCriteria.errorRate", 85.5),
+            JMESPathCheck("autoStopCriteria.errorRateTimeWindowInSeconds", 120),
+        ]
+        self.cmd(
+            "az load test update "
+            "--test-id {test_id} "
+            "--load-test-resource {load_test_resource} "
+            '--load-test-config-file "{load_test_config_file}" '
+            "--resource-group {resource_group} ",
+            checks=checks,
+        )
+        # Invalid autostop test cases
```

Comment:
Couple of more invalid cases

1. time-window less than 0, autostop error rate less than 0
2. time-window in decimal
3. time-window/autostop in string

Reply:
All the invalid cases added
1. autostop not of string type
2. autostop not in allowed values disable/enable
3. autostop error rate > 100.0
4. autostop error rate < 0.0
5. autostop error rate not of number type. This is not needed as the argument is type checked
6. autostop error rate time window < 0
7. autostop error rate time window not of integer type. This is not needed as the argument is type checked
8. autostop from config random string
9. autostop from config error rate > 100.0
10. autostop from config time window < 0